src/MyEventEmitter.js

```diff
@@ -1,12 +1,85 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, listener) {
+    if (typeof listener !== 'function') {
+      throw new Error('Listener must be a function!');
+    }
+
+    this.events[eventName] = this.events[eventName] || [];
+    this.events[eventName].push(listener);
+
+    return this;
+  }
+
+  once(eventName, listener) {
+    const wrapper = () => {
+      listener();
+      this.off(eventName, wrapper);
+    };
+
+    this.events[eventName].push(wrapper);
+
+    return this;
+  }
+
+  off(eventName, listenerToRemove) {
+    if (!this.events[eventName]) {
```

Comment:
I am not sure, that in this case we should throw an error

Reply:
we shouldn't throw error here